paddle/framework/executor.cc

```diff
@@ -116,6 +115,7 @@ void Executor::Run(const ProgramDesc& pdesc, Scope* scope, int block_id,
   for (auto& op_desc : block.AllOps()) {
     auto op = paddle::framework::OpRegistry::CreateOp(*op_desc);
     VLOG(3) << op->DebugStringEx(local_scope);
+    VLOG(3) << "Memory used " << memory::memory_usage(place_);
```

Comment:
Do we need a little more details in this log message, e.g., “before ....” or “after ...”?

Reply:
This PR mainly provides a demo to show the result and these logs are actually for debugging. And another clean PR focus on memory optimization transpiler https://github.com/PaddlePaddle/Paddle/pull/7356 has been merged. 
So I will close this PR.